packages/rs-drive-abci/src/abci/handler/mod.rs

```diff
@@ -683,22 +669,7 @@ where
             });
         };
 
-        let platform_version = block_execution_context
-            .block_platform_state()
-            .current_platform_version()?;
-
-        let got: withdrawal_txs::v0::WithdrawalTxs = vote_extensions.into();
-        let expected = block_execution_context
-            .withdrawal_transactions()
-            .keys()
-            .map(|tx_id| proto::ExtendVoteExtension {
-                r#type: VoteExtensionType::ThresholdRecover as i32,
-                extension: tx_id.to_byte_array().to_vec(),
-                sign_request_id: None,
-            })
-            .collect::<Vec<_>>()
-            .into();
-
+        // TODO: Is it possible to get a message from a fake proposer?
```

Comment:
Please close all TODOs

And no, in theory you should not get proposal from a fake proposer.